tools/SetupFlow/DevHome.SetupFlow/Services/WindowsPackageManager.cs

```diff
@@ -167,6 +162,27 @@ public async Task<bool> StartAppInstallerUpdateAsync()
         }
     }
 
+    public async Task<bool> RegisterAppInstallerAsync()
```

Comment:
Does the `bool` signify if the AppInstaller is registered?

Reply:
True means `AppInstaller` was registered successfully